stylus/objects/layouts.styl

```diff
@@ -91,7 +92,8 @@ $app
         display block
 
         main,
-        main > [role=contentinfo]
+        main > [role=contentinfo], // Deprecated
```

Comment:
ping @y-lohse @goldoraf seems cozy-photos is using this.

Reply:
Actually almost every app uses this, that's why I'm just commenting the deprecated status and not removing it completely yet.

Reply:
Is it enough to just remove the `[role=contentinfo]` ? If so that would be a quick change to do everywhere.

Reply:
It's more replacing than removing but basically yeah, that's it.